solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -1471,6 +1485,16 @@ contract LiquidityProtection is TokenHandler, ContractRegistryClient, Reentrancy
         return 0;
     }
 
+    /**
+     * @dev returns the larger of two values
+     *
+     * @param _val1 the first value
+     * @param _val2 the second value
+     */
+    function max(uint256 _val1, uint256 _val2) internal pure returns (uint256) {
```

Comment:
this should be in Math.sol

Reply:
True, but we will have no `min` function there.

Reply:
and why is that an issue? we don't need `min` right now

Reply:
Fixed